website/docs/concepts/basic-web-technologies/html.mdx

```diff
@@ -7,82 +7,79 @@ comment: 'Keep this file as short and simple as possible. Its purpose is to ease
 import Tabs from '@theme/Tabs'
 import TabItem from '@theme/TabItem'
 
-> Yew mostly operates on the idea of keeping everything that a reusable piece of
-> UI may need, in one place - rust files. But also seeks to stay close to the
-> original look of the technology.
+> Yew centrally operates on the idea of keeping everything that a reusable piece of
+> UI may need in one place - rust files, while also keeping the underlying technology
+> accessible where necessary.
 
 For HTML we achieve this by providing the `html!` macro:
 
 ```rust
 use yew::prelude::*;
 
-let my_header: Html = html!{<img src="img_girl.jpg" alt="Girl in a jacket" width="500" height="600" />};
+let my_header: Html = html!{
+    <img src="img_girl.jpg" alt="Girl in a jacket" width="500" height="600" />
+};
 ```
 
-`html!` macro allows you to write html generation as if it was in a `.html` file and then behind the scenes by yew gets turned into internal builder patterns that generate all DOM nodes.
+The `html!` macro allows you to write html generation as if it was in a `.html` file and
+behind the scenes yew turns it into a rust expression representing the DOM to generate.
 
-As it is part of rust code it provides a easy way to switch from html to rust by applying curly brackets:
+Similar to format expressions, there is an easy way to embed values from the surrounding
+context into the html by applying curly brackets:
 
 ```rust
 use yew::prelude::*;
 
 let header_text = "Hello world".to_string();
-let header_html: Html = html!{<h1>{header_text}</h1>};
+let header_html: Html = html!{
+    <h1>{header_text}</h1>
+};
 
 let count: usize = 5;
-let counter_html: Html = html!{<p>{"My age is: "}{count}</p>};
-
-let combined_html: Html = html!{<div>{header_html}{counter_html}</div>};
-```
-
-One rule major rule comes with use of `html!` - you can only return 1 wrapping node:
-
-<Tabs>
-<TabItem value="Valid" label="Valid">
-
-```rust
-use yew::html;
-
-html! {
-    <div>
-        <div></div>
-        <p></p>
-    </div>
+let counter_html: Html = html!{
+    <p>{"My age is: "}{count}</p>
 };
 
+let combined_html: Html = html!{
+    <div>{header_html}{counter_html}</div>
```

Comment:
```suggestion
let combined_html: Html = html! {
    <div>{ header_html } { counter_html }</div>
```
Just some formatting

Reply:
I've added a space behind `html!` but the other spaces might suggest spaces in the generated HTML, so I'd leave them out.